### PR TITLE
Small product updates

### PR DESCRIPTION
**Pull request by kodster28**
Addresses PCX-3630 and finishes PCX-3522

### CODE REVIEW

File: content/ssl/ssl-for-saas/plans.md [hunk 1]
```diff
@@ -116,3 +123,9 @@ meta:
 </table>
 
 {{</table-wrap>}}
+
+## Enterprise plan benefits
```
Comment (kodster28) [inline content/ssl/ssl-for-saas/plans.md:127]: This is pure pre-sales copy... ugh... we're going to migrate this page away soon, but until we do it needs to be here.

File: content/dns/manage-dns-records/reference/ttl.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+pcx-content-type: reference
+title: Time to Live (TTL)
+weight: 3
+---
+
+# Time to Live (TTL)
+
+**Time to Live (TTL)** is a field on [DNS records](/dns/manage-dns-records/how-to/create-dns-records/) that controls how long each record is valid and — as a result — how long it takes for record updates to reach your end users.
+
+Longer TTLs speed up [DNS lookups](https://www.cloudflare.com/learning/dns/what-is-dns/) by increasing the chance of cached results, but they also mean that updates to your records take longer to go into effect.
```
Comment (crwaters16) [inline content/dns/manage-dns-records/reference/ttl.md]: unclear whether "they" refers to TTLs or cached results

File: content/ssl/edge-certificates/additional-options/certificate-signing-requests.md [hunk 3]
```diff
@@ -41,7 +41,9 @@ To use a CSR:
...
 
-         {{<Aside type="note">}}
+     {{<Aside type="note">}}You will not see the option to adjust your **Encoding Mode** until after you have created a CSR associated with the specific zone or your account.
+     {{</Aside>}}
 
-    You will not see the option to adjust your **Encoding Mode** until after you have created a CSR associated with the specific zone or your account.
-    {{</Aside>}}
+## Renew a certificate
+
+When you [renew a custom certificate](/ssl/edge-certificates/custom-certificates/renewing/), you can re-use a previously generated CSR.
```
Comment (crwaters16) [inline content/ssl/edge-certificates/additional-options/certificate-signing-requests.md]:
```suggestion
When you [renew a custom certificate](/ssl/edge-certificates/custom-certificates/renewing/), you can reuse a previously generated CSR.
```
```suggestion
When you [renew a custom certificate](/ssl/edge-certificates/custom-certificates/renewing/), you can re-use a previously generated CSR.
```

Comment (crwaters16) [inline content/ssl/edge-certificates/additional-options/certificate-signing-requests.md]:
```suggestion
When you [renew a custom certificate](/ssl/edge-certificates/custom-certificates/renewing/), you can reuse a previously generated CSR.
```
```suggestion
When you [renew a custom certificate](/ssl/edge-certificates/custom-certificates/renewing/), you can re-use a previously generated CSR.
```

File: content/ssl/edge-certificates/custom-certificates/renewing.md [hunk 4]
```diff
@@ -10,6 +10,10 @@ Uploaded Certificates cannot be renewed by Cloudflare. You must ensure that you
 
 Cloudflare automatically sends email notification 30 and 14 days before your custom certificate expires. Email is sent to users who have the SSL/TLS, Administrator, or Super Administrator roles.
 
+{{<Aside type="note">}}
+When renewing a custom certificate, you can re-use a [previously generated CSR](/ssl/edge-certificates/additional-options/certificate-signing-requests/).
```
Comment (crwaters16) [inline content/ssl/edge-certificates/custom-certificates/renewing.md]:
```suggestion
When renewing a custom certificate, you can reuse a [previously generated CSR](/ssl/edge-certificates/additional-options/certificate-signing-requests/).
```
```suggestion
When renewing a custom certificate, you can re-use a [previously generated CSR](/ssl/edge-certificates/additional-options/certificate-signing-requests/).
```

Comment (crwaters16) [inline content/ssl/edge-certificates/custom-certificates/renewing.md]:
```suggestion
When renewing a custom certificate, you can reuse a [previously generated CSR](/ssl/edge-certificates/additional-options/certificate-signing-requests/).
```
```suggestion
When renewing a custom certificate, you can re-use a [previously generated CSR](/ssl/edge-certificates/additional-options/certificate-signing-requests/).
```

**Review by crwaters16 — APPROVED** [review]

File: content/dns/manage-dns-records/reference/ttl.md (hunk 2 above)
Comment (kodster28) [inline content/dns/manage-dns-records/reference/ttl.md]:
```suggestion
Longer TTLs speed up [DNS lookups](https://www.cloudflare.com/learning/dns/what-is-dns/) by increasing the chance of cached results, but a longer TTL also means that updates to your records take longer to go into effect.
```

**Pull request merged**